source/mpi_shared_linear_elasticity.cpp

```diff
@@ -480,8 +486,15 @@ namespace Solid
                           quad_stress[i][j][q] = tmp_stress[i][j];
                         }
                     }
+                    ele_stress[0] += (0.25*quad_stress[0][0][q]);
+	                  ele_stress[1] += (0.25*quad_stress[0][1][q]);
+	                  ele_stress[2] += (0.25*quad_stress[1][0][q]);
+	                  ele_stress[3] += (0.25*quad_stress[1][1][q]);
```

Comment:
Please format before commit. Indentation is messed up.
```bash
cd OpenIFEM
clang-format -i -style=file source/*.cpp
clang-format -i -style=file include/*.h
```

Reply:
done